### DISCUSSION TITLE
[FR] Read the Ollama model list and switch quickly.

### DISCUSSION THREAD

**Discussion by wwjCMP** [logancyang/obsidian-copilot#549] [Q&A]
Load the Ollama model list and switch quickly to avoid restarting the plugin.

**Comment by logancyang** [comment]
What do you mean by restarting the plugin? You should be able to change Ollama model name in settings, restart ollama server without restarting the plugin.

**Comment by wwjCMP** [comment]
> What do you mean by restarting the plugin? You should be able to change Ollama model name in settings, restart ollama server without restarting the plugin.

avoid reloading the plugin

**Comment by wwjCMP** [comment]
Quite a few settings should be placed in the dialog window and allowed to be modified at any time without reloading the plugin, such as the number of contexts.
